core/app/modules/kit-library/assets/js/components/item-header.js

```diff
@@ -102,10 +104,22 @@ export default function ItemHeader( props ) {
 		},
 	);
 
-	const applyButton = useKitCallToActionButton( props.model, {
+	const applyButton = useKitCallToActionButton( props.model, props.pageId, {
```

Comment:
Do we need to pass the pageId if you're using it below?

Reply:
Correct